docs/fundamentals/connection/connection-options.txt

```diff
@@ -349,3 +350,40 @@ item, as shown in the following example:
 
 See the `$driverOptions: array <https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBClient__construct/#parameters>`__
 section of the {+php-library+} documentation for a list of driver options.
+
+.. _laravel-disable-id-alias:
+
+Disable Use of id Field Name Conversion
+---------------------------------------
+
+Starting in {+odm-long+} v5.0, ``id`` is an alias for the ``_id`` field
+in MongoDB documents, and the library automatically converts ``id``
+to ``_id`` for both top level and embedded fields when querying and
+storing data.
+
+When using {+odm-long+} v5.3 or later, you can disable the automatic
+conversion of ``id`` to ``_id`` for embedded documents. To do so,
+perform either of the following actions:
```

Comment:
Should we indicate that this option is there to ensure backward compatibility with existing projects, but that its use is not recommended for new projects?